videoencoder.cpp

```diff
@@ -347,6 +338,14 @@ main (int argc, char *argv[])
 
 	  rgb_to_yuv (&image, ycbcr);
 
+	std::string new_filename = filename;
+
+	new_filename = new_filename.replace(new_filename.end() - 4, new_filename.end(), ".png");
```

Comment:
you can just append .png - so if we switch to .jpeg you don't have to hunt why you end up with ..png :)